requirements/examples.txt

```diff
@@ -1,2 +1,2 @@
 torchvision>=0.4.1,<0.9.0
-gym>=0.17.0
\ No newline at end of file
+gym>=0.17.0
```

Comment:
gym doesn't seem to be needed for this example.

Reply:
Gym wasn't added by this PR, i think its just a newline at the end of the file, which we can get rid of :)

Reply:
it is needed in other examples, right?

Reply:
yeah probably, just to clear up i mean that we could get rid of the newline (dnt think its that big a deal) not gym

Reply:
I re-enable these remaining examples in #4551